internal/cnab/driver.go

```diff
@@ -131,3 +134,31 @@ func SetupDriver(installation *store2.Installation, dockerCli command.Cli, opts
 	driverImpl, errBuf := prepareDriver(dockerCli, bind, stdout)
 	return driverImpl, errBuf, nil
 }
+
+func SetRelocationMapFunc(installation *store.Installation) func(op *driver.Operation) error {
```

Comment:
nit: most of this kind of option functor are prefixed by `With`, like `WithRelocationMap`.
So when you use it:
```
 a.Run(&installation.Claim, nil, cnab.WithRelocationMap(installation), WithSomethingElse(...)...)
```

Reply:
✅